frontend/src/components/layout/Layout.module.scss

```
@@ -47,6 +46,10 @@ $toastify-toast-close-button: var(--toast-close-button);
       }
     }
 
+    & > [class*="toast-body"] {
+      padding: 16px;
+    }
+
```

nit: use Protocol token `$spacing-md`

```suggestion
    & > [class*="toast-body"] {
      padding: $spacing-md;
    }

```

---

done